source/form.html

```diff
@@ -40,134 +40,181 @@
+            </label>
           </p>
-          <p class="feedback-form__patronymic">
-            <label class="feedback-form__label" for="visitors-patronymic">ОТЧЕСТВО:</label>
-            <input class="feedback-form__input" type="text" id="visitors-patronymic" name="visitors-patronymic" placeholder="Викторович">
+          <p class="form-group__item">
+            <label class="form-group__field field">
+              Отчество:
+              <input class="field__input" type="text" value="Викторович" name="visitors-patronymic"
+                placeholder="Отчество">
+            </label>
           </p>
         </fieldset>
-
-        <fieldset class="feedback-form__group">
-          <legend class="feedback-form__title">Контактная информация:</legend>
-          <p class="feedback-form__phone">
-            <label class="feedback-form__label" for="contact-phone">Контактный телефон*:</label>
-            <input class="feedback-form__input" type="tel" id="contact-phone" name="contact-phone" placeholder="Введите телефон">
+        <fieldset class="order-form__group form-group">
+          <legend class="form-group__heading">
+            Контактная информация:
+          </legend>
+          <p class="form-group__item">
+            <label class="form-group__field field">
+              Контактный телефон*:
+              <input class="field__input" type="tel" name="contact-phone" placeholder="Введите телефон" required>
+            </label>
           </p>
-          <p class="feedback-form__email">
-            <label class="feedback-form__label" for="email">Электронная почта*:</label>
-            <input class="feedback-form__input" type="email" id="email" name="email" placeholder="Введите e-mail">
+          <p class="form-group__item">
+            <label class="form-group__field field" for="email">
+              Электронная почта*:
+              <input class="field__input" type="email" id="email" name="email" placeholder="Введите e-mail" required>
+            </label>
           </p>
         </fieldset>
-
-
-
-        <fieldset class="feedback-form__group">
-          <legend class="feedback-form__title">Ваше общее впечатление:</legend>
-          <ul class="feedback-form__list">
-            <li class="feedback-form__item">
-              <label class="control">
-                <input class="control-input" type="radio" name="Скорее положительное" checked>Скорее положительное
+        <fieldset class="order-form__group form-impression">
+          <legend class="form-impression__heading">
+            Ваше общее впечатление:
+          </legend>
+          <ul class="form-impression__list">
+            <li class="form-impression__item">
+              <label class="form-impression__field radio">
+                <input class="radio-input" type="radio" name="overall-impression" value="Rather-positive" checked>
```

Comment:
Почему-то лишний БЭМ-блок создается, достаточно и элемента

```suggestion
                <input class="radio__input" type="radio" name="overall-impression" value="Rather-positive" checked>
```